src/main/java/com/stripe/model/Plan.java

```diff
@@ -98,96 +98,44 @@ public void setProductObject(Product product) {
   }
   // </editor-fold>
 
-  public List<PlanTier> getTiers() {
-    return tiers;
-  }
-
-  public void setTiers(List<PlanTier> tiers) {
-    this.tiers = tiers;
-  }
-
-  public String getTiersMode() {
-    return tiersMode;
-  }
-
-  public void setTiersMode(String tiersMode) {
-    this.tiersMode = tiersMode;
-  }
-
-  public PlanTransformUsage getTransformUsage() {
-    return transformUsage;
-  }
-
-  public void setTransformUsage(PlanTransformUsage transformUsage) {
-    this.transformUsage = transformUsage;
-  }
-
-  public String getUsageType() {
-    return usageType;
-  }
-
-  public void setUsageType(String usageType) {
-    this.usageType = usageType;
-  }
```

Comment:
All these accessors were unnecessary as they can be generated by Lombok. This is why the PR has a negative net line count.